src/instructlab/model/serve.py

```diff
@@ -134,6 +134,6 @@ def serve(
         # Run the llama server
         backend_instance.run()
 
-    except ServerException as exc:
+    except llama_cpp.ServerException as exc:
```

Comment:
Why do we need to change this line?

Reply:
@leseb this was a fix for pylint warning. But I see another PR fixes in another way. I'll revert this change by removing `llama_cpp.`

Reply:
It's still present, I thought we agreed on reverting this? Thanks

Reply:
@leseb this version is outdated. It has been removed.